### ISSUE TITLE
What is the Bolt CMS ethos?

### ISSUE THREAD

**Issue by tolusonaike** [bolt/bolt#954]
I came across bolt while trying to pick up Silex and i think its awesome. I hope i don't come off as critical, confrontational, or demanding, i am just really excited about this project.

After reading the docs and some open/closed issues, i get this feeling that there is a "developer" guard protecting its functionalities.  I wanted to understand Bolt's philosophy by posing a few questions:
- Does Bolt have a hard-line against a UI for configuration files? (Since this will be a brick wall for casual users. Period.) 
- Will there ever be ability to customise or add Global settings to the backoffice (Site logo, Description, theme options and settings, etc)
- Will there ever be the ability to manipulate content through "blocks" or widgets through the backoffice

In other words, is Bolt aiming to be THE developer's CMS and nothing else? I know "keep it simple" is a big moto around here...but simple for who?
